api/libproject/devstart.rb

```diff
@@ -1665,6 +1668,8 @@ def deploy(cmd_name, args)
     versions_file = must_get_env_value(gcc.project, :cdr_versions_json)
     update_cdr_versions_for_project("config/#{versions_file}", op.opts.dry_run)
 
+    common.run_inline %W{gradle loadDataDictionary -PappArgs=#{op.opts.dry_run ? true : false}}
```

Comment:
I'm not clear on the difference between `deploy()` and `dev-up` and why dev-up uses docker and deploy does not.

Reply:
I believe it does run in docker actually. The first line of `deploy()` is `ensure_docker cmd_name, args` which will run the rest of the function in a docker environment.